pdns/tkey.cc

```diff
@@ -69,9 +69,9 @@ void PacketHandler::tkeyHandler(const DNSPacket& p, std::unique_ptr<DNSPacket>&
       {
       tkey_out->d_error = 21; // BADALGO
 #ifdef ENABLE_GSS_TSIG
-      g_log<<Logger::Error<<"GSS-TSIG request but feature not enabled by enable-gss-tsigs setting"<<endl;
+      g_log<<Logger::Debug<<"GSS-TSIG request but feature not enabled by enable-gss-tsigs setting"<<endl;
```

Comment:
```suggestion
      g_log<<Logger::Debug<<"GSS-TSIG request but feature not enabled by enable-gss-tsig setting"<<endl;
```

(older typo, might as well fix it now)